README.md

```diff
@@ -14,143 +14,115 @@ The "target system" element is either generic, i.e. **x86_64** or **AArch64**, o
 We used this target to inform our developmental direction and as a more leading-edge installer option. 
 We are preserving this profile for when our in-process Python 2 to 3 move is complete.
+Note however that it is not currently maintained.
+Please contribute to this profile if you are interested as we would very much like to again be available in the Tumbleweeds.
 
 ### Special mention:
-- Leap15.2.ARM64EFI
+- ARM64EFI
 
 We are very exited to welcome contributions for the innovative [Traverse Ten64](https://www.crowdsupply.com/traverse-technologies/ten64) AArch64 platform.
 [Traverse technologies](https://traverse.com.au/) are an important contributor to the [rockstor-core](https://github.com/rockstor/rockstor-core) code base
 and have been instrumental in achieving our initial AArch64 aims.
-Although this profile includes Ten64 specific drivers (inactive on non Ten64 platforms), it is otherwise a modern generic Arm64 installer.
-The resulting installer is intended to supports 64-bit ARM systems that implement the [Embedded Boot](https://github.com/ARM-software/ebbr) or [Server boot](https://github.com/ARM-software/sbsa-acs) standard. 
+The resulting installer is intended to supports 64-bit ARM systems that implement the [Embedded Boot](https://github.com/ARM-software/ebbr) or [Server boot](https://github.com/ARM-software/sbsa-acs) standard.
+Note that additional drives may be required for your specific hardware, if so consider [Installing the Stable Kernel Backport](https://rockstor.com/docs/howtos/stable_kernel_backport.html).
+Also see the following subsection for enabling these same repositories/facilities within the resulting installer itself.
+
+### Stable Kernel Backport & matching btrfs-progs
+Note that we have, remarked out, the 'Stable Kernel Backport' and 'filesystems' repositories within our rockstor.kiwi config.
+This enables building a custom installer with these back-ported modifications out-of-the-box.
+See [Installing the Stable Kernel Backport](https://rockstor.com/docs/howtos/stable_kernel_backport.html) for more information.
+Note that this is not a supported configuration so do indicate this modification if reporting issues on our forum or GitHub.
 
 ## HOWTO
 
 Please see the [overview](https://osinside.github.io/kiwi/overview.html) section of the kiwi-ng docs for the canonical
-[System Requirements](https://osinside.github.io/kiwi/overview.html#system-requirements): e.g. 15 GB free space, Python 3.5+ etc.
+[System Requirements](https://osinside.github.io/kiwi/overview.html#system-requirements) for building the installer: e.g. 15 GB free space, Python 3.5+ etc.
 
-Given our image target OS is exclusively 'Built on openSUSE', a vanilla openSUSE Leap 15.2 install is recommended as the kiwi-ng host operating system.
+Given our image target OS is exclusively 'Built on openSUSE', a vanilla openSUSE Leap 15.3 install is recommended as the kiwi-ng host operating system.
+But if using the build
```

Comment:
@phillxnet, is part missing here? It seems to end abruptly and I'm unsure of what it could be. Sorry if I'm missing something obvious here.

Reply:
@FroggyFlox Cheers. i'll see to that or remove it. Must have gotten distracted mid sentence.
Cheers.

Reply:
@FroggyFlox I remembered where I was going with this and ended up leaping ahead to that and forgot to return. Silly me.
Now fixed via another commit.

Thanks again for the double check and apologies for such a silly mistake. Probably a bit rushed on this one.
We can revisit the ordering throughout this how doc actually but I'll push to master for now given it's prececessor is now way out of date and consequently inadvertently misleading, at least now we have 4.1.0-0 out the door.